pandas/plotting/_core.py

```diff
@@ -3031,19 +3031,46 @@ def barh(self, x=None, y=None, **kwds):
 
     def box(self, by=None, **kwds):
         r"""
-        Boxplot
+        Make a box plot of the DataFrame columns.
+
+        This method draws a box and whisker for each series in the DataFrame
+        in a single chart. The boxes share the vertical axis, which
+        makes it easier to compare if they are similar in scale.
+        Box plots are useful because they clearly show the median,
+        quartiles and flier points in the data.
```

Comment:
There is also the PR on the `boxplot` method: https://github.com/pandas-dev/pandas/pull/20152. It has also extended the summary, you can check if there are some additions over there you could also use here (eg I think the wikipedia link is interesting) 

(ideally we would re-use this extended summary for both, but that is something we can fix later)

Reply:
I've tried to fix the issues that you commented.

The summary from #20152 is more explanatory so I've copied most of it here. I removed the mention to the `by` parameter which, unless I'm missing something, is not yet implemented for plot.box. Is someone working on that?